common/src/types/tasks/descriptors/mod.rs

```diff
@@ -215,6 +215,9 @@ pub mod mocks {
         NewWalletTaskDescriptor, QueuedTask, QueuedTaskState, TaskDescriptor, TaskQueueKey,
     };
 
+    /// The name of a mock matching pool used for testing task history
+    pub const MOCK_MATCHING_POOL: &str = "mock-matching-pool";
```

Comment:
nit: let's move this into the `mocks` module or otherwise somewhere gated behind the "mocks" feature

Reply:
this is already gated behind the mocks feature, and honestly i'm not sure what a better place for it would be, so i'm partial to keeping it here